src/main/kotlin/app/nush/bot/main.kt

```diff
@@ -22,6 +22,10 @@ val helpText = """
     
     **`${config.botPrefix}projects create`**
     Creates new project (admin only)
+    Append "channel-only" if repo already exists
+    
+    **`${config.botPrefix}projects linkrepo`**
```

Comment:
Do add an indication of required arguments. eg ``**`${config.botPrefix}projects linkrepo <repo name>`**``